RELEASE-NOTES.md

```diff
@@ -9,6 +9,9 @@
 * Renamed `SerializerFactory::newSnaksSerializer` to `SerializerFactory::newSnakListSerializer`, leaving a b/c alias
 * Renamed `DeserializerFactory::newClaimDeserializer` to `DeserializerFactory::newStatementDeserializer`, leaving a b/c alias
 * Renamed `DeserializerFactory::newSnaksDeserializer` to `DeserializerFactory::newSnakListDeserializer`, leaving a b/c alias
+* Added `SerializerFactory::newStatementListSerializer`
+* Added `DeserializerFactory::newStatementListDeserializer`
+* Added `DeserializerFactory::newFingerprintDeserializer`
```

Comment:
I think both should not be exposed. There currently is no need for it as far as I can see.